### PR TITLE
Reverts the space nerf PR and adds some light story to space ruins with plushies included

### PR DESCRIPTION
**Pull request by Aphast**
## About The Pull Request
Rebalances all previously changed space loot from ruins as well as migrates plushies and some ruins gain a small bit of implied story

## Why It's Good For The Game
due to[ PR 981 ](https://github.com/Skyrat-SS13/Skyrat-tg/pull/981)space has become an entirely tedious and barely rewarding place with no place to even upgrade your things, no hardsuits or rewards for beating large combat oriented ruins such as the trade route, the only place to even aquire a weapon in space to fight the more combat oriented ruins currently is at the crashed ship by beating a turret to death in which you get a no-pin cannon. and as per some requests some more light story elements were added to the ruins, and of course the plushies that many love, and if the random spawn is ported ill update this with such. however the main point of this pr is to fix the loot changes that were overnerfed during pr 981 of october

## Changelog
🆑/
add: replaces all of the eva suits across every ruin with their previous two piece color counterpart in the derelict or their previous hardsuit equivalent
balance: makes pirates drop their weapons again to allow the trade route to be completed without additional aid from foreign firepower
add: re-adds all previous ballistic weapons to the trade route
add: plushies have migrated across the space ruins making one of each plushie have its home in a selected ruin fitting its theme
add: adds blood and a paper to the gas ruin
add: adds a honkbot to the clown champion planet -HONK-
add: gives the gondola asteroid a suit and mask (no more slaughtering gondolas for a costume.... unless?)
add: gives the receptionists at the space hotel some coffee and tea aswell as a paper and pen to check people into the hotel with
add: makes the hotel a little more romantic with some candles for its tables
add: due to complaints the hotel security is reinforced with a secbot assembly, seems the hotel doesn't cover the full cost of a bot
add: adds spraytan for thoose in the hotel wishing to get darker skin without bathing in the stars radiation
add: re-adds the turreted outposts chest rig and shield
add: re-adds the wt-rifle to deep storage's armory with its previous mags
add: syndicates deserve bedsheets too, gives some sheets to the turreted outposts and a plushie for the softies among them
add: gives the whiteship_box a floorbot
add: makes ceres whiteship fir its previous abductor theme with some tables and a returned suit
add: makes the space bus more modern with some cigarettes and paper strewn about, damn kids even rusted away a good toolbox into an old one
add: the previous occupants of the crashed_ship gained one new appropriately placed gibbed human
add: replaces a useless snipermag in the emptyshell with an equally but slightly less so useless syndicate cosmetic loot drop
add: adds a lesser magic mirror to the space hotel for those wishing for some more rp potential with it
add: the space hotel has become the space goatel without people as a new goat joins the kitchen, welcome gete to the party and dont feed him too much trash or corprate will get mad again
remove: removes all previous loot changes by pull request 281 regarding non-pool items
/:cl:

### CODE REVIEW

**Comment by Oscari-pog** [comment]
The sniper rifle mag can actually be decon-analyzed for illegal tech. Otherwise, wonderful PR.

**Comment by Aphast** [comment]
> The sniper rifle mag can actually be decon-analyzed for illegal tech. Otherwise, wonderful PR.

i do believe i have tested it before and it claims it to be regular ammo so no illegal

**Comment by dawsonkeyes** [comment]
pretty good, replacing old consistent space ruin loot with the newer random loot drops that were added in the nerf PR is nothing but an improvement

File: _maps/RandomRuins/SpaceRuins/spacehotel.dmm [hunk 1]
```diff
@@ -1659,6 +1663,7 @@
 /obj/machinery/atmospherics/components/unary/vent_scrubber/on{
 	dir = 1
 	},
+/obj/structure/mirror/magic/lesser,
```
Comment (Elli-Skala) [inline _maps/RandomRuins/SpaceRuins/spacehotel.dmm]: the magic mirror shouldnt be added due to the custom ghost role pr that makes this just redundant

Comment (Aphast) [reply _maps/RandomRuins/SpaceRuins/spacehotel.dmm]: this was made before the pr so thats why it is there, will fix

**Comment by Gandalf2k15** [comment]
Howdy! So this hasn't been done modularly and will cause us many headaches down the line, what I suggest is adding new ruins, and we can disable these in the config. So it loads your alternative ones! That way we don't really have any issues in the future!

**Pull request closed without merging**